### DISCUSSION TITLE
Default module after a (new) user logs in

### DISCUSSION THREAD

**Discussion by hanisaf** [directus/directus#17029] [Archive]
I developed a custom Directus module that I want to be the default entry point for users after logging in. I noticed that (1) the user is always directed to the last module they were at in the previous session, and (2) that new users are defaulted to the content module.

- is it possible to override (1)? the user is always directed to a specific module after logging in.
- is it possible to override (2)? A new user is directed to a specific module rather than the content module.

Thanks

**Comment by rijkvanzanten** [comment]
The default page the users sees on login is not currently configurable. Directus relies on `directus_users.last_page` for this behavior, so you could override that value 👍🏻
